Challo/Challo/Modules/Map/MapModule.swift

```diff
@@ -9,9 +9,22 @@ import SwiftUI
 
 final class MapModule: ViperModuleProtocol {
     func assemble() -> (view: AnyView, presenter: MapPresenter) {
-        let interactor = MapInteractor()
+        let mapStore = MapStore(mapMarkerRepository: MapMarkerRepository(),
+                                mapRouteRepository: MapRouteRepository(),
+                                mapItineraryRepository: MapItineraryRepository())
+        
+        // TODO make url for alamofiremanager injected in
+        let networkManager = APINetwork.getNetworkManager()
+        if let alamofireManager = networkManager as? AlamofireManager {
+            alamofireManager.apiPath = ProcessInfo.processInfo.environment["google_places_path"]
+        }
+        
+        let placesAPI = PlacesAPI(parser: PlacesAPIParser(), networkManager: networkManager)
```

Comment:
I think now we register the APIs with the SWInject `APIContainer` and then inject the APIs into the different modules in `MainContainerRouter` right?

Reply:
okay let me change it!